model/project.go

```diff
@@ -1217,20 +1217,6 @@ func (p *Project) FindTaskGroup(name string) *TaskGroup {
 	return nil
 }
 
-// FindContainerFromProject finds the container configuration associated with a given task's Container field.
```

Comment:
It's not being called anywhere. I can add it back in if you think we have a use for it in the future.

Reply:
Hmm, we might need that in the future but I suppose it can be removed since we don't have any use cases for retrieving a full `Container` object for a given task at the moment.